components/Shipping/models.rst

```diff
@@ -0,0 +1,186 @@
+Models
+======
+
+Rule
+----
+
+**Rule** object represents additional restrictions which shippable object have to fulfil in order to be support by given
```

Comment:
`A **Rule** object represents additional restrictions which have to be fulfilled by a shippable object in order to be supported by a given **ShippingMethod**. `